frontend/src/lib/lemon-ui/LemonFileInput/LemonFileInput.tsx

```diff
@@ -18,6 +18,8 @@ export interface LemonFileInputProps extends Pick<HTMLInputElement, 'multiple' |
      * styling is applied to the alternativeDropTargetRef
      * **/
     alternativeDropTargetRef?: RefObject<HTMLElement>
+    // the text to display to the user, a sensible default is used if not provided
```

Comment:
If you use the JSDoc notation `/** */`, code editors will know to associate this as a docstring – makes for a handy hover tooltip

Reply:
<img width="645" alt="Screenshot 2024-01-04 at 22 31 03" src="https://github.com/PostHog/posthog/assets/984817/0bd52b08-dc19-4915-9c9f-a4e6fff59968">
ah works in pycharm with either kind... but quick and safe to change 😊

Reply:
Good to know, so it's VS Code's JS language server being pedantic 😄